app.js

```diff
@@ -0,0 +1,245 @@
+      })
+
+      var education = document.getElementById("education")
+      resume.education.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "volunteer" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            education.appendChild(element)
+          }
+      })
+
+      var awards = document.getElementById("awards")
+      resume.awards.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "awards" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            awards.appendChild(element)
+          }
+      })
+      var publications = document.getElementById("publications")
+      resume.publications.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "publications" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            publications.appendChild(element)
+          }
+      })
+
+      var skills = document.getElementById("skills")
+      resume.skills.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "skills" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            skills.appendChild(element)
+          }
+      })
+  
+      var languages = document.getElementById("languages")
+      resume.languages.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "languages" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            languages.appendChild(element)
+          }
+      })     
+
+      var interests = document.getElementById("interests")
+      resume.languages.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "interests" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            interests.appendChild(element)
+          }
+      })     
+
+
+       var projects = document.getElementById("projects")
+       resume.projects.forEach (elementArr => {
+        for (const [key, value] of Object.entries(elementArr)) {
+            className = "projects" + key
+            element = document.createElement("P")
+            element.textContent = value
+            element.classList.add( className )
+            projects.appendChild(element)
+          }
+      })     

```

Comment:
There is a lot of repetition in your code. You could make this into one reusable function:

```
function displayData (property) {
       var projects = document.getElementById(property)
       resume[property]forEach (elementArr => {
        for (const [key, value] of Object.entries(elementArr)) {
            className = property + key
            element = document.createElement("P")
            element.textContent = value
            element.classList.add( className )
            projects.appendChild(element)
          }
      })
}
```

Then call it for `work` through to `projects`.
e.g. displayData("work");